provider/application_resource.go

```diff
@@ -83,6 +83,39 @@ func applicationResource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"permissions": &schema.Schema{
```

Comment:
Does this needs to be in provider ?

Reply:
This is a configuration in the application component, we will need to support it. So, we need to map the field. Please, check the other changes made in our internal module.